__tests__/app.test.js

```diff
@@ -57,10 +57,30 @@ describe("GET/api/categories", () => {
         });
     });
   })
+  describe("GET /api/reviews/:review_id", () => {
+    test("should return a review object with the specified properties", () => {
+      return request(app)
+        .get("/api/reviews/2")
+        .expect(200)
+        .then(({ body }) => {
+          expect(body).toEqual({
```

Comment:
Should use either `.toMatchObject` or `expect.ObjectContaining` here to ensure we allow for extensibility (more properties present in the future)